### PR TITLE
feat: adding google calendar integration to events page

### PR DESCRIPTION
**Pull request by nikvoblikov**
**This PR adding Events page**
TODO: 

- [x] Develop main structure and UI;
- [x] Add filtering;
- [x] Add gatsby-source-google-calendar plugin;
- [ ] Add link to CardWithCta
- [ ] Integrate eCHO events.
- [ ] Change Google Calendar data in Netlify

**Steps to test**

- Open the [page](https://deploy-preview-379--ebpfio.netlify.app/events)
- Confirm that everything looks and works as expected

### CODE REVIEW

**Comment by xmulligan** [comment]
We would like to have the conferences drop down to be a list of "All events, eBPF Summit, Cloud native eBPF Day, eBPF Track (LPC), and bpfconf (LSF/MM/BPF)". each one of those should also be in the event filter and clicking on it should bring up a list of all of the years of it

File: gatsby-config.js [hunk 1]
```diff
@@ -44,14 +55,22 @@ module.exports = {
...
     'gatsby-plugin-image',
     'gatsby-transformer-sharp',
     {
       resolve: 'gatsby-plugin-sharp',
       options: {
         defaults: {
           quality: 85,
-          placeholder: 'none',
+          placeholder: 'dominantColor',
+          transformOptions: { cropFocus: 'attention' },
```
Comment (vannyle) [inline gatsby-config.js:73]: I'm not sure if we need the placeholder for the whole website. I would suggest adding the placeholder to the specific image, where you would like it.

File: src/components/pages/events/filters/dropdown-with-two-select/dropdown-with-two-select.jsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+import React, { useRef, useState } from 'react';
+
+import Checkbox from 'components/shared/checkbox';
+import useClickOutside from 'hooks/use-click-outside';
+import ChevronIcon from 'icons/chevron.inline.svg';
+
+const DropdownWithTwoSelect = ({
```
Comment (vannyle) [inline src/components/pages/events/filters/dropdown-with-two-select/dropdown-with-two-select.jsx:9]: The naming is a bit confusing to me, can we rename it to dropdown-with-two-levels?

File: src/components/pages/events/filters/dropdown-with-two-select/dropdown-with-two-select.jsx [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+          <ul className="flex flex-col">
+            {itemsMain.map(({ name }, index) => (
+              <li className="flex items-center space-x-2.5 pt-2.5 first:pt-0 md:pt-3" key={index}>
+                <Checkbox
+                  id={name}
+                  name="type"
+                  label={name}
+                  value={name}
+                  checked={valuesMain.includes(name)}
+                  onChange={(e) => handleSelect(e.target.checked, e.target.value, mainFilter)}
```
Comment (vannyle) [inline src/components/pages/events/filters/dropdown-with-two-select/dropdown-with-two-select.jsx:88]: should we pass event to the function, the get state and value on function level? 
```suggestion
                  onChange={(e) => handleSelect(e, mainFilter)}
```

File: src/components/pages/events/hero/hero.jsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import ListCard from './list-card';
+import MainCard from './main-card';
+
+const title =
+  'Participate in eBPF events that highlight</br>a range of expert voices and inspiring ideas';
```
Comment (vannyle) [inline src/components/pages/events/hero/hero.jsx:8]: I would hardcode the title instead of putting in the variable

Comment (vannyle) [inline src/components/pages/events/hero/hero.jsx:8]:
```suggestion

```

File: content/events/README.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
+# Blog
```
Comment (vannyle) [inline content/events/README.md:1]:
```suggestion
# Events
```

**Review by vannyle — APPROVED** [review]

**Review by vannyle — APPROVED** [review]

**Comment by xmulligan** [comment]
is this still being worked on?

**Pull request closed without merging**